src/main/java/com/nukkitx/nbt/IntegerInternPool.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Adapted from The MIT License (MIT)
```

Comment:
Should probably follow the project license instead of separate license for this file?